crates/goose-mcp/src/developer/mod.rs

```diff
@@ -1387,8 +1387,10 @@ impl DeveloperRouter {
             )
         })?;
 
```

Comment:
xcap 0.0.14 -> xcap 0.7.0

-    pub fn title(&self) -> &str {
-        &self.impl_window.title
-    }
-    /// The window process id
-    pub fn pid(&self) -> u32 {
-        self.impl_window.pid
+    pub fn title(&self) -> XCapResult<String> {
+        self.impl_window.title()
+    }